### PR TITLE
Process queue Django Command

### PR DESCRIPTION
**Pull request by itaybleier**
@Schweigi - 
* Should there be exception handling on the `_process_message()` function?
* Do you see any issue with listening on multiple queues in this manner?

### CODE REVIEW

File: eb_sqs/management/commands/process_queue.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    def add_arguments(self, parser):
+        parser.add_argument('--queues', '-q',
+                            dest='queue_names',
+                            help='Name of queues to process, separated by commas')
+
+    def handle(self, *args, **options):
+        if not options['queue_names']:
+            raise CommandError('Queue names (--queues) not specified')
+
+        queue_names = options['queue_names'].split(',')
```
Comment (Schweigi) [inline eb_sqs/management/commands/process_queue.py]: Would be great to remove white spaces

```
queue_names = [queue_name.trim() for queue_name in options['queue_names'].split(',')]
```

File: eb_sqs/management/commands/process_queue.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                for msg in messages:
+                    logger.debug('Read message {}'.format(msg.message_id))
+                    self._process_message(msg)
+                    logger.debug('Processed message {}'.format(msg.message_id))
+                    msg.delete()
+                    logger.debug('Deleted message {}'.format(msg.message_id))
+
+    def _process_message(self, message):
+        # type: (Message) -> None
+        worker = WorkerFactory.default().create()
```
Comment (Schweigi) [inline eb_sqs/management/commands/process_queue.py]: Create worker before while loop to re-use

File: eb_sqs/settings.py [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 from django.conf import settings
 
+AWS_REGION = getattr(settings, 'AWS_REGION', 'us-east-1')  # type: unicode
+
+MAX_NUMBER_OF_MESSAGES = getattr(settings, 'EB_SQS_MAX_NUMBER_OF_MESSAGES', 10)
```
Comment (Schweigi) [inline eb_sqs/settings.py]: Type hints missing

Comment (Schweigi) [inline eb_sqs/settings.py]: New parameters not documented in readme

File: eb_sqs/management/commands/process_queue.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+from django.core.management import BaseCommand, CommandError
+
+from eb_sqs import settings
+from eb_sqs.worker.worker_factory import WorkerFactory
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
```
Comment (Schweigi) [inline eb_sqs/management/commands/process_queue.py]: Add small section to readme on how to use.

File: eb_sqs/management/commands/process_queue.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        while True:
+            for queue in queues:
+                messages = queue.receive_messages(
+                    MaxNumberOfMessages=settings.MAX_NUMBER_OF_MESSAGES,
+                    WaitTimeSeconds=settings.WAIT_TIME_S,
+                )
+
+                for msg in messages:
+                    logger.debug('Read message {}'.format(msg.message_id))
+                    self._process_message(msg)
```
Comment (Schweigi) [inline eb_sqs/management/commands/process_queue.py]: Exception during message processing will cause the worker to stop. Best to add exception handling in _process_message.

**Comment by itaybleier** [comment]
@Schweigi - I updated the PR according to your comments.

Regarding the scale issue- I think it's fine that this `process_queue` command will be able to work with multiple queues. The problem of scaling it is up to the caller- one option is to use separate task definitions for each queue.

**Pull request merged**